bip-0085.mediawiki

```diff
@@ -360,36 +360,58 @@ OUTPUT
 * DERIVED ENTROPY=f7cfe56f63dca2490f65fcbf9ee63dcd85d18f751b6b5e1c1b8733af6459c904a75e82b4a22efff9b9e69de2144b293aa8714319a054b6cb55826a8e51425209
 * DERIVED PWD=_s`{TW89)i4`
 
-===RSA===
+===GPG Keys===
 
-Application number: 828365'
+Application number is dependant on the key type.
```

Comment:
I would recommend a single application number and then extend the path or different key types. That's more consistent with the rest of BIP85:
```
{same_app_number_for_all}/{key_type}/{key_bits}/{key_index}
```

Reply:
This would be the neatest solution